communications/meetings.md

```diff
@@ -0,0 +1,67 @@
+
+Guide:
+
+- Send a calendar invite to all participants, with Zoom/Hangouts Meet.
+- Create the meeting agenda on Hugo for the meeting, provide sufficient time ahead of the meeting to accommodate all participants.
+    - For a net-new meeting, the organizer(s) should define the initial scope of agenda items.
+    - For a recurring meeting with past sessions, use the agenda/notes items from the previous meeting session as a starting point. Remove ones no longer needed or already resolved.
+- Initiate one or more meeting note(s) on Hugo for meeting agenda discussions.
+    - Send these notes to the relevant Slack channel.
+    - Edit meeting agenda to reflect the outcome of these discussions.
+- In each meeting note, participants write down briefing, information, knowledge, and ideas that are relevant.
+    - For a recurring meeting with past sessions, focus on briefings of pending items from the previous meeting session, such as what each _has done_, _has been doing_, and _plans to do_. Give everyone else a sense of _where you are_ and _the direction you're heading to_. This helps others to determine if there are potential directional conflicts/blocks.
+    - Raise relevant questions, problems, obstacles, unknowns, and any other issues. Try to make them demonstratable, by leveraging modern tools such as screen-captured images or videos, and/or steps to reproduce these issues.
+    - Preemptively discuss. Seek to resolve as many issues and directional conflicts/blocks as possible.
+- Cross off resolved (or portions of) items from the list of the meeting agenda. In the event of having all the item discussions resolved, skip to [section: After A Meeting](#after-a-meeting).
+- Put a tentative timebox on each unresolved item. This can be an outcome of a collective agreement.
+
+Tips:
+
+Turn off Hugo filter on "Hide meetings with notes" to fit the practice of having pre-meeting notes for discussions.
+
+<img width="323" alt="notes filter" src="https://user-images.githubusercontent.com/2837532/76785553-0a0e1980-678c-11ea-8878-ece9c06c930e.png">
+
+Since Hugo does a good job at syncing notes discussion as Slack threads, liberally participate on either platform.
+
+<img width="562" alt="sync" src="https://user-images.githubusercontent.com/2837532/76794286-0d5dd100-679d-11ea-80b9-2077e2c6ce3f.png">
+
+## During A Meeting
+
+Objectives:
+- Align & inform all participants of relevant information at the same time (synchronously)
+- Maintain time for each agenda item & scope of meeting as intended
+- Record as many details as possible for future context/meetings
+- Crowdsource bookkeeping effort to increase accuracy, and minimize bias.
+
+Guide:
+- Briefly go over the agenda items, and timebox given for each item.
+- Demonstrate issues, instead of abstractly talking about them.
+- If certain items cannot be resolved within the timebox, record all reasons and context into the notes formed in [section: Before A Meeting](#before-a-meeting) and move on.
```

Comment:
should be 'record all reasons and context into the note titled Meeting Notes formed in ...'

Reply:
no, same as the other point

> my intention is to actually keep using the per-item notes to be used during meeting. i may need to further clarify/reword to emphasize that.